lib/bundles/inspec-habitat/profile.rb

```diff
@@ -92,6 +106,20 @@ def verify_profile
       Habitat::Log.info('Profile is valid.')
     end
 
+    def vendor_profile_dependencies
```

Comment:
It feels like we're duplicating code here: https://github.com/chef/inspec/blob/master/lib/inspec/base_cli.rb#L149 Can we extract one method to meet the needs of `base_cli` and this implementation?

This `vendor_deps` is also used https://github.com/chef/inspec/blob/master/lib/bundles/inspec-compliance/cli.rb#L160